docs/guides/Fulfillment/setting-up-white-label-seller-as-pickup-point.md

```diff
@@ -0,0 +1,44 @@
+- [Loading dock](https://help.vtex.com/en/tutorial/managing-loading-docks--7K3FultD8I2cuuA6iyGEiW): configure a loading dock and link it to the shipping policy associated with the pickup point. You can use the [Create/update dock](https://developers.vtex.com/docs/api-reference/logistics-api#post-/api/logistics/pvt/configuration/docks) endpoint.
+- [Warehouse](https://help.vtex.com/en/tutorial/managing-warehouses--tutorials_137): you must configure a warehouse and link it to the shipping policy associated with the pickup point. You can use the [Create/update warehouse](https://developers.vtex.com/docs/api-reference/logistics-api#post-/api/logistics/pvt/configuration/warehouses) endpoint.
+
+## Fill in inventory quantity
+
+After setting the pickup point, the shipping policy, the loading dock, and the warehouse, fill in the quantity of the items in inventory. You can also use the [Update inventory by SKU and warehouse](https://developers.vtex.com/docs/api-reference/logistics-api#put-/api/logistics/pvt/inventory/skus/-skuId-/warehouses/-warehouseId-) endpoint.
+
+The franchise account does not have its own catalogue, it inherits products and SKU information from the main account, so it is necessary to update quantity through the [import and export of the inventory spreadsheet](https://help.vtex.com/en/tutorial/importing-and-exporting-an-inventory-spreadsheet--tutorials_2034).
+
+## Validate configuration
+
+When you made the pickup point configurations correctly in the franchise account, SKUs with inventory and price can be sold on the main account. The franchise account will work as white label seller and the main account will work as marketplace.
+
+>❗ Make sure your white label seller is active. In your VTEX Admin, go to __Marketplace > Sellers > Management__, and in the seller row in column _Status_, click `Active`.
+
+### White label seller and franchise account
+
+To check if the white label seller is delivering a SKU through the registered pickup point, use the [Cart simulation endpoint](https://developers.vtex.com/docs/api-reference/checkout-api#post-/api/checkout/pub/orderForms/simulation).
+
+When the configuration was made correctly, the call will return the shipping information of the a given SKU. That is enough to check if the shipping is occuring by the registered pickup point.
+
+### Marketplace and main account
+
+To check if the Marketplace is including the SKU of the white label seller in the shopping cart, use the [Cart simulation endpoint](https://developers.vtex.com/docs/api-reference/checkout-api#post-/api/checkout/pub/orderForms/simulation).
+
+The call should return shopping cart information to where the SKU was sent in the context of the request body.
```

Comment:
Essa frase me parece bem confusa. 
"where the SKU was sent"  = o endereço físico do pick up point, ou sent pro catálogo da main account? 

E que contexto do request body? Sem exemplo de código fica extremamente difícil de entender o que essa frase quer dizer.

Reply:
Alterei a frase.